storefront/settings.py

```
@@ -167,3 +174,39 @@
+    'disable_existing_loggers' : False,
+    'handlers':  {
+        'file' : {
+            'class' : 'logging.FileHandler',
+            'filename' : 'general.log',
+            'formatter' : 'verbose'
+        }
+    },
+    'loggers': {
+        '': {
+            'handlers' : ['file'],
+            'level': os.environ.get('DJANGO_LOG_LEVEL' , 'INFO')
```

Ruff (0.8.2)

Undefined name `os`

(F821)